### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by kulothunganug**

### CODE REVIEW

File: kivymd/uix/textfield.py [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ def on_line_color_focus(self, *args):
         if self.color_mode == "custom":
             self._update_colors(self.line_color_focus)
 
-    def on__hint_text(self, instance, value):
+    def on_hint_text(self, instance, value):
```
Comment (HeaTTheatR) [inline kivymd/uix/textfield.py:1039]: This is not a typo - https://github.com/kivy/kivy/blob/master/kivy/uix/textinput.py#L2633

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/textfield.py (hunk 1 above)
Comment (kulothunganug) [reply kivymd/uix/textfield.py:1039]: Does it a mistake in typing or it's already correct?

Comment (HeaTTheatR) [reply kivymd/uix/textfield.py:1039]: This is an overridden built-in method of the Kivy library textinput widget.

Comment (kulothunganug) [reply kivymd/uix/textfield.py:1039]: :+1:

**Pull request closed without merging**